### ISSUE TITLE
Masonry Layout has problem when we check-uncheck gutter checkbox in full width layout.

### ISSUE THREAD

**Issue by inkcolor** [Obox/layerswp#372]
Masonry Layout has problem when we uncheck gutter checkbox.
and wide layout option with masonry with and without gutter has problem, it not taking margin from left and extra margin from right.

Tested on fresh copy of 1.5.5

[img]http://i.share.pho.to/e683aa0b_o.png[/img]
[img]http://i.share.pho.to/1319f131_o.png[/img]

**Comment by vailjoy** [comment]
Please confirm the following:

1. Your images are all larger than 600px wide
2. Your Settings > Media Medium image is set to 600or higher width 
3. Thumbnails  have been regenerated since any media changes
4. All plugins are disabled
5. http://docs.layerswp.com/doc/how-to-center-masonry-columns-or-add-equal-space-on-each-side/

**Comment by inkcolor** [comment]
Yes, i have tested it in fresh installation with 1000X1200 px images, but still checking it with different ways to find it that working properly or not just because i think i will be layers fan and want to make it perfect and thanks for great support :) I am new for layers and trying to release my first themeforest item asap in layers.

**Comment by inkcolor** [comment]
also when we check gutter space and as its set 20px not taking that right gutter between blocks.

**Comment by inkcolor** [comment]
My testing is continue on masonry layout and i am coming with new one look like a issue.
please take look on screenshot which i prepare, when selecting column setting
6 of 12 column
3 of 12 column
3 of 12 column
3 of 12 column
3 of 12 column

http://i.share.pho.to/dd31f10a_o.png
